src/main/kotlin/uk/gov/justice/digital/hmpps/manageoffencesapi/repository/OffenceRepository.kt

```diff
@@ -46,11 +46,9 @@ interface OffenceRepository : JpaRepository<Offence, Long> {
     "SELECT o FROM Offence o WHERE " +
       "LOWER(o.code) LIKE LOWER(CONCAT(:prefix1, '%')) " +
       "OR LOWER(o.code) LIKE LOWER(CONCAT(:prefix2, '%')) " +
-      "OR LOWER(o.code) LIKE LOWER(CONCAT(:prefix3, '%')) " +
-      "OR LOWER(o.code) LIKE LOWER(CONCAT(:prefix4, '%')) " +
       "",
   )
-  fun findByCodeStartsWithAnyIgnoreCase(prefix1: String, prefix2: String, prefix3: String, prefix4: String): List<Offence>
```

Comment:
the prefix lookup for SA00001 and SA00002 is not necessary as the have been moved into their own schedule